internal/indexer/indexer.go

```diff
@@ -456,16 +518,20 @@ func (i *Indexer) indexDefinition(p *packages.Package, filename string, document
 		}))
 	}
 
-	if _, ok := obj.(*types.PkgName); ok {
-		i.emitImportMoniker(resultSetID, p, obj)
-	}
+	// @eric, I don't think we want this to happen anymore
+	// if _, ok := obj.(*types.PkgName); ok {
+	// 	i.emitImportMoniker(resultSetID, p, obj)
+	// }
 
 	if obj.Exported() {
 		i.emitExportMoniker(resultSetID, p, obj)
 	}
 
 	// If the pkg/object has associated documentation, link to it. This enables e.g. going from documentation
 	// for a symbol <-> its definition/hover/references/etc in either direction.
+	//
+	// @eric I'm not sure we need to even do this anymore.
+	// We have the monikers correct now, so I don't think there is any reason to even do anything for these (maybe?)
```

Comment:
Let's keep as much in-index as we can for now until we're sure we have sufficient index coverage and can rely on x-repo lookups of documentation/hover text.

Currently our schema is better suited for in-index queries, as the other path has additional database interactions.